machine_learning_hep/processer.py

```diff
@@ -405,25 +410,18 @@ def process_efficiency(self):
                                n_bins, analysis_bin_lims)
             h_sel_fd = TH1F("h_sel_fd" + stringbin2, "FD Reco and sel in acc |#eta|<0.8 and sel", \
                             n_bins, analysis_bin_lims)
-            h_gen_pr = TH1F("h_gen_pr" + stringbin2, "Prompt Generated in acceptance |y|<0.5", \
```

Comment:
@ginnocen  What would be the reason for defining everything twice? Should something else happen here or was this just some copy paste thing?